Platform/ArrowlakeBoardPkg/Script/StitchIfwiConfig_arlh.py

```diff
@@ -154,8 +154,8 @@ def get_xml_change_list (platform, plt_params_list):
         ('./FlashLayout/BiosRegion/InputFile',                                                      '$SourceDir\BiosRegion.bin'),
         ('./FlashLayout/Ifwi_IntelMePmcRegion/MeRegionFile',                                        '$SourceDir\ME Sub Partition.bin'),
         ('./FlashLayout/Ifwi_IntelMePmcRegion/PmcBinary',                                           '$SourceDir\CsePlugin#PMC.bin'),
-        ('./FlashLayout/EcRegion/EcRegionPointer',                                                  '$SourceDir\DescriptorPlugin#EcRegionPointer.bin'),
-        ('./FlashLayout/EcRegion/InputFile',                                                        '$SourceDir\SystemPlugin#EcRegion.bin'),
+        ('./FlashLayout/EcRegion/EcRegionPointer',                                                  '$SourceDir\DescriptorPlugin#EcRegionPointer_arlh.bin'),
+        ('./FlashLayout/EcRegion/InputFile',                                                        '$SourceDir\SystemPlugin#EcRegion_arlh.bin'),
```

Comment:
ARL-U and ARL-H ingredients are in different folder. I don't see any reason to rename them. and need update the release note if we rename them.

Reply:
We will help to update the release note and BIOS IFWI already combine the u/h release.

Reply:
All platforms use the last node name as the file name except ARL. And it is easier to use existing tool to get these ingredients from IFWI and manage them.  do we need align with other platforms?

Reply:
check with ee lim and ec and acm have different content in u/h.

Reply:
> All platforms use the last node name as the file name except ARL. And it is easier to use existing tool to get these ingredients from IFWI and manage them. do we need align with other platforms?
Customer will follow the release note to get the IFWI ingredients, rename to target mapping name documented in release note. so the file name not sync to mfit decompose result I think it's ok.